.github/workflows/main.yaml

```diff
@@ -129,9 +136,69 @@ jobs:
       - name: Build Wasmer binary
         run: |
           make build-wasmer
-          # TODO: build wapm
-          # make build-wapm
+      - name: Build Wapm binary
+        run: |
+          make build-wapm
+        if: needs.setup.outputs.DOING_RELEASE == '1'
+      - name: Package Wasmer
+        run: |
           make package
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v2
+        if: needs.setup.outputs.DOING_RELEASE == '1'
+        with:
+          name: wasmer-${{ matrix.os }}
+          path: dist
+
+  release:
+    needs: [setup, test]
+    runs-on: ubuntu-latest
+    if: needs.setup.outputs.DOING_RELEASE == '1'
+    steps:
+      - name: Download The Artifacts
```

Comment:
```suggestion
      - name: Download the Artifacts
```

Reply:
I was thinking of this as a title and generally `the` isn't capitalized in titles

https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html

Reply:
Oh, that's what you're suggesting, I misread this as the opposite. 👍 for the fix